paper/paper.md

```diff
@@ -35,21 +35,22 @@ bibliography: paper.bib
 
-It is written in Python and has Infrastructure as Code (IaC) deployment that can be used to quickly and repeatably deploy it to Azure.
+RCTab is designed to be flexible and extensible and can be easily adapted to the needs of different organisations. It is written in Python and has Infrastructure as Code (IaC) deployment that can be used to quickly and repeatably deploy it to Azure.
 
 ## Statement of Need
 
-Institutions are increasingly adopting cloud platforms for both operational and research computing infrastructure, with many adopting hybrid or cloud-first approaches. The benefits include easy and flexible access to cutting edge hardware without substantial upfront investment.
+Commercial cloud services, such as Amazon Web Services, Microsoft Azure, Google Cloud Platform, and others, offer users flexible and scalable computing and storage resources, in addition to a variety of services and unique solutions. Due to their flexibility and convenient on-demand access to these resources, commercial cloud services are increasingly becoming the preferred platform for both academic and commercial users. They are especially suited for handling heterogeneous workloads, enabling cloud bursting, or even serving as components of a hybrid cloud solution where the cloud complements on-premises resources.
 
-Compared to traditional infrastructure with dedicated servers (e.g. on-premises/HPC systems), this presents a new challenge in controlling costs, particularly for the latest hardware such as the GPUs. Therefore, it is important to have robust cost management mechanisms.
+On-demand access can present challenges for organisations that require a degree of predictability and stability in their cost planning. This is because the cost of cloud services can vary from month to month, and some providers do not offer convenient ways to set hard limits on the amount of money that can be spent or the duration until which the allocated budget is valid. This is especially true for organisations with a large number of users, such as research institutes, and can lead to significant unexpected costs, if the service is not monitored closely. To address this, organisations typically either limit the number of users accessing resources and the type of resources they can access, or employ dedicated staff to monitor resource usage and costs. Neither approach is ideal: the former can restrict the cloud's potential, while the latter can be time-consuming and prone to errors.
```

Comment:
```suggestion
On-demand access can present challenges for organisations that require a degree of certainty in their outgoings. This is because the cost of cloud services can vary by region and over time, and some providers do not offer convenient ways to set hard limits on the amount of money or duration for which an allocated budget is valid. This is especially true for organisations with a large number of users, such as research institutes, and can lead to significant unexpected costs if the service is not monitored closely. To address this, organisations typically either limit the number of users accessing resources and the type of resources they can access, or employ dedicated staff to monitor resource usage and costs. Neither approach is ideal: the former can restrict the cloud's potential, while the latter can be time-consuming and prone to errors.
```